command/pr_checkout.go

```diff
@@ -57,41 +57,44 @@ func prCheckout(cmd *cobra.Command, args []string) error {
 		headRemote, _ = remotes.FindByRepo(pr.HeadRepositoryOwner.Login, pr.HeadRepository.Name)
 	}
 
+	localBranchName, err := cmd.Flags().GetString("branch")
+	if err != nil {
+		return err
+	}
+	if localBranchName == "" {
+		localBranchName = fmt.Sprintf("pr/%d", pr.Number)
+	}
+
 	var cmdQueue [][]string
-	newBranchName := pr.HeadRefName
 	if headRemote != nil {
 		// there is an existing git remote for PR head
-		remoteBranch := fmt.Sprintf("%s/%s", headRemote.Name, pr.HeadRefName)
-		refSpec := fmt.Sprintf("+refs/heads/%s:refs/remotes/%s", pr.HeadRefName, remoteBranch)
+		srcRef := fmt.Sprintf("refs/heads/%s", pr.HeadRefName)
+		destRef := fmt.Sprintf("refs/remotes/%s/%s", headRemote.Name, pr.HeadRefName)
+		refSpec := fmt.Sprintf("+%s:%s", srcRef, destRef)
 
 		cmdQueue = append(cmdQueue, []string{"git", "fetch", headRemote.Name, refSpec})
 
-		// local branch already exists
-		if _, err := git.ShowRefs("refs/heads/" + newBranchName); err == nil {
-			cmdQueue = append(cmdQueue, []string{"git", "checkout", newBranchName})
-			cmdQueue = append(cmdQueue, []string{"git", "merge", "--ff-only", fmt.Sprintf("refs/remotes/%s", remoteBranch)})
+		if _, err := git.ShowRefs("refs/heads/" + localBranchName); err == nil {
+			// local branch already exists
+			cmdQueue = append(cmdQueue, []string{"git", "checkout", localBranchName})
+			cmdQueue = append(cmdQueue, []string{"git", "merge", "--ff-only", destRef})
 		} else {
-			cmdQueue = append(cmdQueue, []string{"git", "checkout", "-b", newBranchName, "--no-track", remoteBranch})
-			cmdQueue = append(cmdQueue, []string{"git", "config", fmt.Sprintf("branch.%s.remote", newBranchName), headRemote.Name})
-			cmdQueue = append(cmdQueue, []string{"git", "config", fmt.Sprintf("branch.%s.merge", newBranchName), "refs/heads/" + pr.HeadRefName})
+			cmdQueue = append(cmdQueue, []string{"git", "checkout", "-b", localBranchName, "--no-track", destRef})
+			cmdQueue = append(cmdQueue, []string{"git", "config", fmt.Sprintf("branch.%s.remote", localBranchName), headRemote.Name})
+			cmdQueue = append(cmdQueue, []string{"git", "config", fmt.Sprintf("branch.%s.merge", localBranchName), srcRef})
 		}
 	} else {
 		// no git remote for PR head
 
-		// avoid naming the new branch the same as the default branch
-		if newBranchName == pr.HeadRepository.DefaultBranchRef.Name {
-			newBranchName = fmt.Sprintf("%s/%s", pr.HeadRepositoryOwner.Login, newBranchName)
-		}
-
```

Comment:
Removed this, because we are now using `pr/<num>` as the default branch, so we don't need to check for this